__tests__/app.test.js

```diff
@@ -52,6 +52,48 @@ describe('getTopics', () => {
   });
 });
 
+describe('postTopic', () => {
+  test('202 /api/topics allows user to create a new topic', () => {
```

Comment:
202 is an incorrect status for this scenario, unfortunately. What you want is 201 (created).